build/python37/jaraco/build.sh

```diff
@@ -17,21 +17,34 @@
 
 . ../../../lib/functions.sh
 
-PKG=library/python-3/jaraco.functools-37
-PROG=jaraco.functools
-VER=2.0
-SUMMARY="jaraco.functools - Additional functools"
-DESC="$SUMMARY"
+PKG=library/python-3/jaraco-37
+PROG=jaraco
+VER=1.0.0
+SUMMARY="jaraco"
+DESC="A bundle of jaraco python modules required by cherrypy"
```

Comment:
I know this is the reason for this package, but I suggest dropping "required by cherrypy"